src/services/account-service.js

```diff
@@ -12,6 +12,13 @@ export class AccountListService {
     });
   }
 
+  static remove(index) {
```

Comment:
please rename this method to `del` to be consistent with what we have in transactions service